src/lib/utils/toc.ts

```diff
@@ -3,11 +3,12 @@ import type { SourceHeadingItem, ToCItem } from "@/lib/types"
 
 // RegEx patterns
 const customIdRegEx = /^.+(\s*\{#([^\}]+?)\}\s*)$/
-const emojiRegEx = /<Emoji [^/]+\/>/g
```

Comment:
No longer needed since compiled source passes this using `mdx(Emoji, ...)` syntax